CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing
+
+When contributing to this repository, please check [Issues](https://github.com/storm-devs/storm-engine/issues) and
+[Projects](https://github.com/storm-devs/storm-engine/projects),
+but you will probably start a [discussion](https://github.com/storm-devs/storm-engine/discussions) first.
```

Comment:
I'd also mention the discord channel here:

> When contributing to this repository, please check [Issues](https://github.com/storm-devs/storm-engine/issues) and
[Projects](https://github.com/storm-devs/storm-engine/projects). Also, it's a good idea to drop a message on our [Discord channel](https://discord.gg/jmwETPGFRe) or start a [discussion](https://github.com/storm-devs/storm-engine/discussions) first to let others know what you're planning to work on.

Reply:
I think we should leave a link to contributing.md in that section